src/Microsoft.DotNet.Tools.Compiler/Program.cs

```diff
@@ -586,14 +628,31 @@ private static bool PrintSummary(List<DiagnosticMessage> diagnostics, Stopwatch
 
             Reporter.Output.WriteLine();
 
-            Reporter.Output.WriteLine($"Time elapsed {sw.Elapsed}");
+            if (preconditions.Present())
```

Comment:
Good idea. I'll have Preconditions return the message while PrintSummary prints it with style. This way I won't couple Preconditions to Reporter.